api/lib/reducers/terraV3-reducers.js

```diff
@@ -25,24 +23,7 @@ function undelegationEndTimeReducer(transaction) {
   }
 }
 
-function delegationReducer(delegation, validator, active, network) {
-  const coinLookup = network.getCoinLookup(network, delegation.balance.denom)
-  const { amount, denom } = coinReducer(delegation.balance, coinLookup)
-
-  return {
-    id: delegation.validator_address.concat(`-${denom}`),
-    validatorAddress: delegation.validator_address,
-    delegatorAddress: delegation.delegator_address,
-    validator,
-    amount,
-    active
-  }
-}
-
 module.exports = {
   ...cosmosV2Reducers,
-  blockReducer: cosmosV3Reducers.blockReducer,
-  setTransactionSuccess: cosmosV3Reducers.setTransactionSuccess,
-  undelegationEndTimeReducer,
-  delegationReducer
+  undelegationEndTimeReducer
 }
```

Comment:
Yeah, this got lost then : shrug: Let me just try very quickly.

Oops too late :sweat_smile: